study-builder/studies/osteo/study.conf

```diff
@@ -1631,6 +1686,36 @@
       "dispatchToHousekeeping": false,
       "order": 3
     },
+    {
+      "label": "CREATE_FH_ON_CONSENT_COMPLETE",
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "FAMILY_HISTORY"
+      }
+      "maxOccurrencesPerUser": 1,
```

Comment:
Should we add a `cancelExpr` here? I'm thinking of the "pediatric age-up" scenario:
* Parent enrolls child, fills out parental consent
* Parental consent completed, creates family history
* Parent completes family history
* Child ages up, get a new self consent and re-enroll in study
* Child completes the self consent, and then this event runs and creates a second family history.

Is that what we're expecting? Should there be 2 family history surveys for aged-up participants?

Reply:
Again, need to follow up for definites from CMI, but in this case I can't imagine two family history surveys is desirable. What does the cancelExpr look like to handle this situation?

Reply:
I think something like this will work fine. You can pick your favorite from [the PEX wiki page](https://broadinstitute.atlassian.net/wiki/spaces/DDP/pages/1109819399/PEX+Language+Guide). I see now that the "top-level" activity has `"maxInstancesPerUser": 1,` which should already cover this case, so maybe we don't need a cancel expression here.
```
"cancelExpr": """user.studies["CMI-OSTEO"].forms["FAMILY_HISTORY"].hasInstance()""",
```

Reply:
@yufengwng Perhaps we don't need a cancelExpr for this scenario? Looks like creation event action checks maxOccurrencesPerUser, so presumably the kid aging up is already a participant and would already have a family history activity?

Reply:
`maxOccurrencesPerUser` will do well for new participants, but won't handle existing ones since this event hasn't ran for them. Either way, I think `maxInstancesPerUser` will do fine here.